### PR TITLE
Use absolute image paths

### LINKED ISSUE #20: Fix image paths
Links to images in [onion client](https://tox-rs.github.io/book/onion/client.html) are broken.

### PR DESCRIPTION
**Pull request by suhr**
Fixes https://github.com/tox-rs/book/issues/20.

This is probably not the best solution, but it should work.

### CODE REVIEW

File: src/onion/client.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This is a description of onion layer in Tox.
 
 Onion routing can be described using this chart:
 
-![](./static/path.svg)
```
Comment (kpp) [inline src/onion/client.md:7]: how about / instead of ./ ?

Comment (suhr) [reply src/onion/client.md:7]: That would resolve to `https://tox-rs.github.io/` instead of `https://tox-rs.github.io/book/`.

Comment (kpp) [reply src/onion/client.md:7]: Did you try?

Comment (suhr) [reply src/onion/client.md:7]: Yes. `/book/static/path.svg` works, as expected.

Comment (kpp) [reply src/onion/client.md:7]:
```diff
-![](https://tox-rs.github.io/book/static/path.svg)
+![](../static/path.svg)
```
This works too

**Pull request merged**